docs/releasing.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Releasing
+weight: 20
```

Comment:
Could this page be moved in a "Development" section so it is not mixed with other user documentation in https://inspektor-gadget.io/docs/latest/contributing/?

Reply:
I moved it to the contributing document. Is it what you intended?

Reply:
Makes sense. I updated the PR.